src/phoenix/server/thread_server.py

```diff
@@ -6,6 +6,9 @@
 from starlette.applications import Starlette
 from uvicorn import Config, Server
 
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.NullHandler())
```

Comment:
Is this use of `NullHandler` to suppress log messages to standard error unless the user explicitly configures logging?